style/sdf/health_cross.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16" height="16" version="1.1" xmlns="http://www.w3.org/2000/svg">
+ <path d="m5 0v5h-5v6h5v5h6v-5h5v-6h-5v-5h-6z" fill-rule="evenodd"/>
```

Comment:
Some maps use a cross, while others use an “H”. Both symbols would be reasonably intuitive to those familiar with American maps. The former is language-agnostic, while the latter is more specific, ruling out doctors’ offices and urgent care facilities.

Reply:
It sounds like maybe hospitals could be an **H** and other health care could be a **+**.